redash/utils/routes.py

```diff
@@ -0,0 +1,15 @@
+from werkzeug.routing import BaseConverter
+
+
+class SlugConverter(BaseConverter):
+    def to_python(self, value):
+        # This is ay workaround for when we enable multi-org and some files are being called by the index rule:
+        # for path in settings.STATIC_ASSETS_PATHS:
+        #     full_path = safe_join(path, value)
+        #     if os.path.isfile(full_path):
+        #         raise ValidationError()
```

Comment:
Considering this is commented out, we might not need this converter at all? 🤔

Reply:
I checked and it's actually the [`to_url`](https://github.com/pallets/werkzeug/blob/03c81ae0e7efbc83b7ea245fb42614795daca0cd/src/werkzeug/routing.py#L1251-L1264) method that would differ. I think it should be fine getting rid of it since it deals with encoding and all other parameters use the default converter anyway. Just didn't mean to break anything.

Reply:
Removed it.